### ISSUE TITLE
Don't trigger F65 if alt missing but title, aria-labelledy, OR aria-label are populated on an image

### ISSUE THREAD

**Issue by DavidMacDonald** [wet-boew/wet-boew-wpss#72]
WCAG recently opened up F65 to  title, aria-labelledy, aria-label so if NONE of those OR alt is populated then the failure is triggered.

I fought against that, but WCAG now allows other ways to provide alternate text on images instead of alt... they read fine on all modern screen readers... they populate the accName in the accessibility API.

**Comment by irwink** [comment]
I will add a check for the other text alternatives for images in the next release.

**Comment by irwink** [comment]
Version 6.0.0 accepts ARIA attributes as text alternatives for images.
